flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/DispatcherFactory.java

```diff
@@ -34,7 +36,9 @@
 	 */
 	Dispatcher createDispatcher(
 		@Nonnull RpcService rpcService,
```

Comment:
No `@Nonnull` I think. We don't mark `@Nonnull` in `SessionDispatcherFactory` though.

Reply:
Yes you are right. Will remove the annotations.